R/extract.R

```diff
@@ -90,11 +90,33 @@ extract.Arima <- function(model, include.pvalues = FALSE, include.aic = TRUE,
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{Arima} objects
+#'
+#' \code{\link{extract}} method for \code{Arima} objects. These objects are
+#' created by the \code{\link[stats]{Arima}} function in the \pkg{stats} package.
+#'
+#' @param model A model object.
+#' @param include.pvalues Report p values in the GOF block?
```

Comment:
Let's try to have a unified spelling in the package. I would opt for `p-values` (with a hyphen).